src/adapters/adapter.js

```diff
@@ -73,10 +103,17 @@ class Adapter {
                 else folders[item.path] = item.children = []
               }
 
-              // encodes but retains the slashes, see #274
-              const encodedPath = path.split('/').map(encodeURIComponent).join('/')
-              item.a_attr = {
-                href: this._getItemHref(repo, type, path)
+              // If item is part of a PR, jump to that file's diff
+              if (item.patch && typeof item.patch.diffId === 'number') {
```

Comment:
Initially I was checking `if (item.patch.diffId)`, but that returns false when `diffId === 0`